packages/flutter/lib/src/material/bottom_navigation_bar.dart

```diff
@@ -164,10 +164,14 @@ class BottomNavigationBar extends StatefulWidget {
   /// former is preferred, [fixedColor] only exists for the sake of
   /// backwards compatibility.
   ///
-  /// The [showSelectedLabels] argument must be non-null.
+  /// If [showSelectedLabels] is `null`, [BottomNavigationBarThemeData.showSelectedLabels]
+  /// is used. If [BottomNavigationBarThemeData.showSelectedLabels]  is null,
+  /// then [showSelectedLabels] defaults to `true`.argument defaults to `true`.
```

Comment:
typo here - remove trailing `argument defaults to `true`.

Reply:
Thanks for spotting this